README.md

```diff
@@ -156,3 +156,11 @@ e.g. the ones verifying the state of each service it depends on.
 ## License
 
 This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details
+
+### Running functional tests
+
```

Comment:
Can you add some overview details here of the tech stack used for automated testing, including codeceptjs, puppeteer etc. Something as a starting point to build upon as the happy path test suite grows.